requirements.txt

```diff
@@ -1,18 +1,17 @@
 # Requirements for production / heroku
 
 # Django
-# 1.8 is waiting on django-simple-history
-Django==1.7.8  # rq.filter: >=1.7, <1.8
+Django==1.7.10  # rq.filter: >=1.7, <1.8
 
 # Better templates
 MarkupSafe==0.23
-Jinja2==2.7.3
+Jinja2==2.8
```

Comment:
There's only a few pages with non-ASCII content, which still work after this change (using public links, tested locally):

https://browsercompat.herokuapp.com/api/v1/maturities/1
https://browsercompat.herokuapp.com/browse/maturities/2
https://browsercompat.herokuapp.com/importer/issues/inline_text (fancy punctuation)
https://browsercompat.herokuapp.com/importer/726 (select alternate language)
https://browsercompat.herokuapp.com/api/v1/view_features/730?format=html&lang=ja

I'll watch for issues when it goes to production.